main.go

```diff
@@ -127,7 +128,15 @@ func compare(args []string) {
 	//schemaPathOld := filepath.Join(basePath, path, "schema-master.json")
 	//schOld := readSchema(schemaPathOld)
 
-	schemaUrlOld := fmt.Sprintf("https://raw.githubusercontent.com/pulumi/pulumi-%s/%s/provider/cmd/pulumi-resource-%[1]s/schema.json", provider, oldCommit)
+	var org string
+	if len(args) == 4 {
```

Comment:
Another option could be passing `myorg/myprovider` as the second argument? Then we could detect the (lack of) `/` and default to `pulumi`.

Reply:
That was my initial idea @mikhailshilkov but didn't want to break the existing workflows, happy to do this if it's preferable?

Reply:
Yes this a better idea, will update